src/main/java/me/zeroeightsix/kami/module/modules/combat/CombatSetting.kt

```diff
@@ -55,11 +63,12 @@ object CombatSetting : Module() {
     private val ignoreOffhandEating = register(Settings.booleanBuilder("IgnoreOffhandEating").withValue(true).withVisibility { page.value == Page.IN_COMBAT && pauseForEating.value })
     private val pauseBaritone = register(Settings.booleanBuilder("PauseBaritone").withValue(true).withVisibility { page.value == Page.IN_COMBAT })
     private val resumeDelay = register(Settings.integerBuilder("ResumeDelay").withRange(1, 10).withValue(3).withVisibility { page.value == Page.IN_COMBAT && pauseBaritone.value })
+    private val motionPrediction = register(Settings.booleanBuilder("MotionPrediction").withValue(true).withVisibility { page.value == Page.IN_COMBAT })
```

Comment:
Is this good enough to be on by default?

Reply:
At least it is reliable :thonkkill: